ports/samd/external_flash_device.h

```diff
@@ -0,0 +1,461 @@
+/*
```

Comment:
If we need to have this file, IMO it should go in a common location like `drivers/memory/`.

Reply:
Sure. Can go there. The file caters for flash chips with different properties.